com.cubrid.cubridmigration.core/src/com/cubrid/cubridmigration/cubrid/meta/CUBRIDSchemaFetcher.java

```diff
@@ -103,6 +110,14 @@ public final class CUBRIDSchemaFetcher extends
 			+ " FROM db_attribute a , db_class c"
 			+ " WHERE c.class_name = a.class_name AND c.class_type='CLASS' AND c.is_system_class='NO' and from_class_name is NULL"
 			+ " ORDER BY a.class_name, c.class_type, a.def_order";
+	
+	private static final String ALL_TABLE_COLUMN_WITH_USERNAME = "SELECT a.class_name, c.owner_name, a.attr_name, a.attr_type, a.from_class_name,"
+			+ " a.data_type, a.prec, a.scale, a.is_nullable,"
+			+ " a.domain_class_name, a.default_value, a.def_order,c.is_reuse_oid_class, c.comment"
```

Comment:
This SQL also seems to need a comment modification.

Reply:
I know. It will be resolved at the next commit